jssource/Minifier.php

```diff
@@ -121,6 +121,12 @@ class Minifier
      */
     static public function minify($js, $options = array())
     {
+        global $sugar_config;
+
+        if(isset($sugar_config['developerMode']) && $sugar_config['developerMode'] === true) {
```

Comment:
Can we change the config variable name for something else, please? - for e.g usually I don't want to switch to developer mode only for testing because I would to see exactly same thing as the end-user but I want to remove the js-compressing to speed-up the slow cache rebuilding process.
Thanks